src/pages/worklist/WorkListTable.tsx

```diff
@@ -82,7 +85,9 @@ const WorklistTable: FC<WorklistTableProps> = ({ publications }) => {
                   <Button
                     color="primary"
                     component={RouterLink}
-                    to={`/publication/${publication.identifier}`}
+                    to={`/publication/${publication.identifier}?tab=${
+                      user.isCurator ? PublicationTab.Submission : PublicationTab.Description
```

Comment:
man må vel være curator for å ha tilgang til worklist, så trenger ikke å sjekke igjen her?

Reply:
godt poeng! fikser